elastic/kibana
# [Fleet] Displaying policy changes in Agent activity

## Summary

Closes https://github.com/elastic/kibana/issues/141206

Added queries to `/action_status` API to query policy updates and query the corresponding revisions of agents.
See more explanation here:
https://github.com/elastic/kibana/issues/141206#issuecomment-1469723504

On UI displaying these `POLICY_CHANGE` actions with some additional info (policy name and new revision).

To test:
- Create agent policies and update them
- See that the policy change appears in agent activity
- Enroll agents to an agent policy and then update the policy
- See that the policy change appears in agent activity, should go through In progress and then Complete state as the Agents receive the new policy change revision.

<img width="528" alt="image" src="https://user-images.githubusercontent.com/90178898/225590483-aa0273c3-df35-42c9-913a-815f130db4c3.png">
<img width="529" alt="image" src="https://user-images.githubusercontent.com/90178898/225592220-4a43a301-6144-452a-94b8-ae059b4a5de9.png">

### Checklist
